10/umbraco-forms/SUMMARY.md

```diff
@@ -17,8 +17,8 @@
   * [Overview Of The Field Types](editor/creating-a-form/fieldtypes/README.md)
     * [Date](editor/creating-a-form/fieldtypes/date.md)
     * [File Upload](editor/creating-a-form/fieldtypes/fileupload.md)
-    * [reCAPTCHA V2](editor/creating-a-form/fieldtypes/recaptcha2.md)
-    * [reCAPTCHA V3](editor/creating-a-form/fieldtypes/recaptcha3.md)
+    * [ReCAPTCHA V2](editor/creating-a-form/fieldtypes/recaptcha2.md)
```

Comment:
I think these were correct as they were - as "reCAPTCHA" is the brand name (see [here](https://www.google.com/recaptcha/about/)).  Not sure what the title is on the page they link to, but that should also use "reCAPTCHA" and not "ReCAPTCHA".